news/this-week-in-fluvio-0065.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "This Week in Fluvio #65"
+date: 2024-09-16
+weight: 20
+---
+Fluvio is a distributed, programmable streaming platform written in Rust.
+
+---
+We released **Fluvio 0.11.12** last week.
+
+## New release
+Fluvio **v0.11.12** is now available!
+
+To update you can run `fvm update`
+
+```bash
+$ fvm update
+
+info: Updating fluvio stable to version 0.11.12. Current version is 0.11.9.
+info: Downloading (1/5): fluvio@0.11.12
+info: Downloading (2/5): fluvio-cloud@0.2.26
+info: Downloading (3/5): fluvio-run@0.11.12
+info: Downloading (4/5): cdk@0.11.12
+info: Downloading (5/5): smdk@0.11.12
+done: Installed fluvio version 0.11.12
+done: Now using fluvio version 0.11.12
+
+```
+
+If you don't have Fluvio in your machine run:
+
+```
+curl -fsS https://hub.infinyon.cloud/install/install.sh | bash
+```
+
+If you are enjoying Fluvio please share with your friends!
+
+## New features
+Notable changes in this new version:
+
+- Now you can use `fluvio cluster upgrade` to upgrade local clusters to the latest Fluvio version.
+- Improvements for the feedback of the SPU-to-SPU connection on mirroring. Also improving the feedback of the failures showing errors.
+
```

Comment:
- `fluvio profile export <profile>` now works with FLV_PROFILE_PATH, thanks to open source contributor @pangan21 for help with this fix, and see docs for how this works.
- `cdk build` now defaults to more compatible native targets

Reply:
added!